### PR TITLE
refactor(x/data): consistent use of content hash

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

Closes: #1006

This pull request applies consistent use of `content_hash` and `content_hashes` in the data module.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by technicallyty — APPROVED** [review]
lgtm

File: proto/regen/data/v1/genesis.proto [hunk 1]
```diff
@@ -15,8 +15,8 @@ message GenesisState {
 
 // GenesisContentEntry is a genesis content entry
 message GenesisContentEntry {
-  // hash is the ContentHash
-  ContentHash hash = 1;
+  // content_hash is the ContentHash
```
Comment (aleem1314) [inline proto/regen/data/v1/genesis.proto:18]: I think we can delete this file. We are not using genesis types.

**Review by aleem1314 — APPROVED** [review]
lgtm

File: proto/regen/data/v1/genesis.proto (hunk 1 above)
Comment (ryanchristo) [reply proto/regen/data/v1/genesis.proto]: https://github.com/regen-network/regen-ledger/pull/1014/commits/700eefcef6155ee55495e36515d1bd0d307be47d

**Pull request merged**